pkgs/kde/gear/kio-extras/default.nix

```
@@ -0,0 +1,43 @@
+{
+  mkKdeDerivation,
+  qt5compat,
+  qtsvg,
+  pkg-config,
+  libkexiv2,
+  samba,
+  libssh,
+  libmtp,
+  libimobiledevice,
+  gperf,
+  libtirpc,
+  openexr,
+  taglib,
+  shared-mime-info,
+  libappimage,
+  kio,
+}:
+mkKdeDerivation {
+  pname = "kio-extras";
+
+  extraNativeBuildInputs = [pkg-config shared-mime-info];
+  extraBuildInputs = [
+    qt5compat
+    qtsvg
+    libkexiv2
+
+    samba
+    libssh
+    libmtp
+    libimobiledevice
+    gperf
+    libtirpc
+    openexr
+    taglib
+    libappimage
+  ];
```

```suggestion
    taglib
    libappimage
    libX11
    libXcursor
  ];
```

See https://github.com/NixOS/nixpkgs/pull/243816

---

Just xcursor is enough, and weird behavior from the CMake file there to not report it, but good find, thanks.

---

Even better. The `find_package(X11)` had me think `libX11` is also needed, but good to know that's not the case.